web/components/TextBlock/SponsorsSection/SponsorsSection.tsx

```diff
@@ -1,33 +1,67 @@
 import { PortableTextComponentProps } from '@portabletext/react/dist/react-portable-text.esm';
-import { Sponsor } from '../../../types/Sponsor';
+import { Sponsor as TSponsor } from '../../../types/Sponsor';
+import { Sponsorship } from '../../../types/Sponsorship';
 import { EntitySectionSelection } from '../../../types/EntitySectionSelection';
-import Sponsors from '../../Sponsors';
+import Sponsor from '../../Sponsor';
+import GridWrapper from '../../GridWrapper';
+import Heading from '../../Heading';
+import styles from './SponsorsSection.module.css';
 
 type SponsorsSectionProps = {
   type: EntitySectionSelection;
-  allSponsors: Sponsor[];
-  sponsors?: Sponsor[];
+  allSponsorships: Sponsorship[];
+  sponsors?: TSponsor[];
 };
 
 export const SponsorsSection = ({
-  value: { type, allSponsors, sponsors },
+  value: { type, allSponsorships, sponsors },
 }: PortableTextComponentProps<SponsorsSectionProps>) => {
-  if (!Array.isArray(allSponsors) || allSponsors.length === 0) {
+  if (!Array.isArray(allSponsorships) || allSponsorships.length === 0) {
     console.error(
-      `SponsorsSection missing or invalid sponsors array: '${allSponsors}'`
+      `SponsorsSection missing or invalid sponsorships array: '${allSponsorships}'`
     );
     return null;
   }
 
-  switch (type) {
-    case 'all':
-      return <Sponsors sponsors={allSponsors} />;
-    case 'highlighted':
-      return <Sponsors sponsors={sponsors} />;
-    case 'none':
-      return null;
-    default:
-      console.error(`Unrecognized SponsorsSection type: '${type}'`);
-      return null;
+  if (type === 'none') {
+    return null;
+  }
+
+  if (type === 'highlighted') {
+    return (
+      <section className={styles.sponsorLevel}>
```

Comment:
I think this is missing a `GridWrapper` around it?

Reply:
[0c37066](https://github.com/sanity-io/structured-content-2022/pull/71/commits/0c370662048462d1730c3426ae6fe4ff782f935e)